README.md

```diff
@@ -67,8 +67,20 @@ contextOptions.Permissions = new[] { ContextPermission.Geolocation };
 
 var context = await browser.NewContextAsync(contextOptions);
 var page = await context.NewPageAsync();
-await page.GoToAsync("https://maps.google.com");
-page.ClickAsync("text='Your location'"); //
+await page.GoToAsync("https://www.google.com/maps");
+
+const string MY_LOCATION_BTN_SELECTOR = ".ml-button-my-location-fab";
+await page.ClickAsync(MY_LOCATION_BTN_SELECTOR);
+
+await page.WaitForLoadStateAsync(LifecycleEvent.Networkidle);
+
+const string STAY_ON_WEB_BTN_SELECTOR = ".ml-promotion-no-thanks";
+if ((await page.QuerySelectorAsync(STAY_ON_WEB_BTN_SELECTOR)) != null)
+{
+    await page.ClickAsync(STAY_ON_WEB_BTN_SELECTOR);
+    await page.WaitForTimeoutAsync(500);
```

Comment:
Let's remove this. I think we don't need this for the purpose of this example.

Reply:
I've removed it. But It was required cause the modal was taking some time to fade out.